tools/make/kube.mk

```diff
@@ -1,11 +1,14 @@
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION ?= 1.24.1
+# GATEWAY_API_VERSION refers to the version of Gateway API CRDs.
+# For more details, see https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/#installing-gateway-api-crds-manually
+GATEWAY_API_VERSION ?= 0.5.0
```

Comment:
We should have a comment in go.mod that states this var should be updated when updating the module or:
```
GATEWAY_API_VERSION=$(go list -m -f '{{.Version}}' sigs.k8s.io/gateway-api)
```

Reply:
nice trick, updating the PR